### PR TITLE
Feature/local storage

### LINKED ISSUE #7: Local Storage
As a Developer,
I need some means to save the progress data,
therefore I need to implement storage

AC:

- [x] Implement LocalStorage

### PR DESCRIPTION
**Pull request by sebuzz**
## Motivation

As a Developer,
I need some means to save the progress data,
therefore I need to implement storage
## Issues closed

closes #7

### CODE REVIEW

File: src/ions/store/store.jsx [hunk 1]
```diff
@@ -1,100 +1,108 @@
...
+						state.currentCard = cardData;
+					})
+				);
+			},
+			shownCards: [],
+			setShownCards: card => {
+				set(
+					produce(state => {
+						if (state.shownCards.find(element => element.cardNumber === card)) {
+							//console.log("multiple found!!!");
```
Comment (konrascher3) [inline src/ions/store/store.jsx:83]: Debug-Comment

File: src/ions/store/store.jsx [hunk 2]
```diff
@@ -1,100 +1,108 @@
...
+							const multiple = state.shownCards.find(
+								element => element.cardNumber === card
+							);
+							multiple.occurrenceSC += 1;
+							//console.log("double.occurrence:", multiple.occurrenceSC);
```
Comment (konrascher3) [inline src/ions/store/store.jsx:88]: See above

File: src/ions/store/store.jsx [hunk 3]
```diff
@@ -1,100 +1,108 @@
...
+							state.currentCard.occurrence = multiple.occurrenceSC;
+						} else {
+							console.log("added new card number");
```
Comment (konrascher3) [inline src/ions/store/store.jsx:91]: And here

File: src/ions/store/store.jsx [hunk 4]
```diff
@@ -1,100 +1,108 @@
...
+							const newCardNumber = {
+								cardNumber: card,
+								occurrenceSC: 1,
+							};
+							state.shownCards.push(newCardNumber);
+							console.log("card:", card);
+						}
+						console.log("=>", JSON.stringify(state.shownCards));
```
Comment (konrascher3) [inline src/ions/store/store.jsx:99]: And here

**Review by konrascher3 — APPROVED** [review]
Hast paar console-Logs drinnen gelassen

File: src/ions/store/store.jsx (hunk 1 above)
Comment (sebuzz) [reply src/ions/store/store.jsx:83]: thank you

**Pull request merged**